xchange-lakebtc/pom.xml

```diff
@@ -25,6 +25,12 @@
 	<dependencies>
 
 		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.3.2</version>
+		</dependency>
```

Comment:
you need to delete the line:

    <version>3.3.2</version>

in `xchange-lakebtc/pom.xml`